Helpers/LicenseHelper.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace K10rProject\Helpers;
+
+use Shopware\Bundle\PluginInstallerBundle\Service\PluginLicenceService;
+use Shopware\Components\Model\ModelManager;
+use Shopware\Models\Plugin\License;
+
+class LicenseHelper
+{
+    /** @var PluginLicenceService */
+    private $licenseService;
+
+    /** @var ModelManager */
+    private $modelManager;
+
+    public function __construct(PluginLicenceService $licenseService, ModelManager $modelManager)
+    {
+        $this->licenseService = $licenseService;
+        $this->modelManager   = $modelManager;
+    }
+
+    /**
+     * Add a license key using SwagLicense
+     *
+     * @param string $licenseKey
+     */
+    public function addLicenseKey($licenseKey)
+    {
+        if (method_exists($this->licenseService, 'importLicence')) { // SW 5.5 compatibility
```

Comment:
Shouldn't we add an alternative or at least a message?

Reply:
This functionality is not used in 5.5 anymore, as there's no code encryption/ionCube present. I can, however, output a deprecation error.

Reply:
Ah! Haven't thought about that 😊Deprecation message is ok